docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -36,7 +36,6 @@ pre[class*="language-"] {
     padding: 1.5rem;
     margin: 1.5rem 0;
     overflow: auto;
-    background-color: var(--color-neutral-50);
```

Comment:
@nzakas this was the only property removed that had a different value, as they both used variables, so I didn't see it as a fallback. The others with unique/fallback values were moved so they're side-by-side and configured to remove that style

Reply:
Gotcha. Thanks for the details.